enterprise/app/workflows/workflows_user.tsx

```diff
@@ -0,0 +1,140 @@
+import React from "react";
+import { User } from "../../../app/auth/auth_service";
+import { OutlinedLinkButton } from "../../../app/components/button/link_button";
+import router from "../../../app/router/router";
+import rpcService, { CancelablePromise } from "../../../app/service/rpc_service";
+import { workflow } from "../../../proto/workflow_ts_proto";
+import WorkflowsZeroStateAnimation from "./zero_state";
+import { GitMerge } from "lucide-react";
+import capabilities from "../../../app/capabilities/capabilities";
+import error_service from "../../../app/errors/error_service";
+import { normalizeRepoURL } from "../../../app/util/git";
+import { Link } from "../../../app/components/link/link";
+import ActionListComponent from "./action_list";
+
+export type WorkflowsUserProps = {
+  user: User;
+};
+
+type State = {
+  workflowHistoryLoading: boolean;
+  workflowHistoryResponse: workflow.GetWorkflowHistoryResponse | null;
+};
+
+export default class WorkflowsUserComponent extends React.Component<WorkflowsUserProps> {
```

Comment:
I feel like it would be simpler to hide the few controls that are admin specific, rather than copying this component code. I think it's just
- The controls section on the RHS of each workflow item
- The "Link a repository" button

wdyt?

Reply:
Honestly, I don't know.  I don't really like forking but I also don't really like showing / hiding.  Maybe it's a bit premature to split things, though..

Another consideration is just what RPCs do we want to have available / to whom.  Right now I'm just taking the results from GetHistory on the developer page and showing those (things that aren't actively running don't show).  Maybe this is just straight bad and we could expose the other two get calls for developers, too?

Reply:
It seems fine to me to allow developers to get linked repositories / workflows, just not modify them. @siggisim wdyt?

Reply:
Alright, just one file it is.